apps/web-giddh/src/app/contact/contact.component.scss

```diff
@@ -526,6 +526,10 @@ button.btn.btn-filter {
 }
 
 .all-table {
+    table {
+        z-index: -1;
+        position: relative;
```

Comment:
use class

Reply:
not available class for z-index: -1;
position DONE